source/libraries/ContractDeployer.ts

```diff
@@ -20,18 +20,26 @@ import {
     WETH9,
     DSToken,
     DSRoles,
-    DSValue
+    DSValue,
+    MatchingMarket,
+    GemPit
 } from "./ContractInterfaces";
+import BN = require("bn.js");
 
 type ContractAddressMapping = { [name: string]: string };
-type NetworkAddressMapping = { [networkId: string]: ContractAddressMapping };
+
+const ETHER = new BN(10).pow(new BN(18));
 
 export class ContractDeployer {
     private readonly accountManager: AccountManager;
     private readonly configuration: DeployerConfiguration;
     private readonly connector: Connector;
     private readonly contracts: Contracts;
 
+    private readonly defaultEthPriceInUsd   = "0x0000000000000000000000000000000000000000000000200000000000000000";
```

Comment:
BTW~ any idea why they chose bytes for this?

Reply:
No.  😢 They also chose bytes for CDP IDs.  I think in that case though it is to stop people from doing exactly what I do and assuming that they are sequential and I can iterate over them.  :p 

In this case it does seem rather silly.